mifosng-provider/src/main/java/org/mifosplatform/organisation/staff/domain/Staff.java

```diff
@@ -196,6 +194,12 @@ public void changeOffice(final Office newOffice) {
             this.joiningDate = newValue.toDate();
         }
 
+        final String imageIdParamName = "imageId";
+        if (command.isChangeInLongParameterNamed(imageIdParamName, this.image.getId())) {
+            final Long newValue = command.longValueOfParameterNamed(officeIdParamName);
```

Comment:
You have a copy/paste bug here. You need the "imageIdParamName" instead of "officeIdParamName"